src/commands/math/commands.js

```diff
@@ -369,11 +373,11 @@ var SummationNotation = P(MathCommand, function(_, super_) {
   };
 });
 
-LatexCmds['∑'] =
+LatexCmds['?'] =
 LatexCmds.sum =
 LatexCmds.summation = bind(SummationNotation,'\\sum ','&sum;');
 
-LatexCmds['∏'] =
+LatexCmds['?'] =
```

Comment:
I did a second pass scan of the cumulative diff, and it seems like this is the only file and https://github.com/sclower/mathquill/commit/f53cfb10caacbc1f9bf3b1d31f5589cabb78a199 is the only commit where unicode symbols got replaced incorrectly.